index.js

```diff
@@ -42,7 +42,7 @@ class Server {
           for(let route of this.#stack) {
             if(route.fast_star || route.regexp != null && (m = route.match(u.pathname))){
                 matched_route = route;
-                req.params = m.params;
+                req.params = m?m.params:null;
```

Comment:
For readability could you format it like this:

```
req.params = m ? m.params : null;

Reply:
I'll just merge this in and then change the formatting